### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by jiaqi20**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -750,28 +751,45 @@ Term | Explanation
...
 **List recipes** | `recipes`
 **Delete recipe** | `deleteR INDEX`<br> e.g. `deleteR 3`
 **Edit recipe** | `editR INDEX [n/NAME] [i/INGREDIENT [ -QUANTITY][, MORE INGREDIENTS [ -QUANTITY]]] [c/CALORIES] [img/IMAGE] [inst/INSTRUCTION[. MORE INSTRUCTIONS]] [t/TAG]...` <br> e.g. `editR 2 n/Apple salad i/apple`
-**Get edit recipe info** | `editR INDEX` <br> e.g. `editR 2`
-**Select recipe**| `selectR INDEX` <br> e.g. `selectR 3`
+**Get edit recipe info** | `editR INDEX`<br> e.g. `editR 2`
+**Select recipe** | `selectR INDEX`<br> e.g. `selectR 3`
 **Close recipe drawer**| `close`
-**Search for recipe** | `searchR [i/INGREDIENT [MORE_INGREDIENT]] [n/NAME [MORE_NAME]] [t/TAG [MORE_TAG]]` <br> e.g. `searchR i/lettuce tomato`, `searchR n/salad`, `searchR t/healthy`
+**Search for recipe** | `searchR [i/INGREDIENT [MORE_INGREDIENT]] [n/NAME [MORE_NAME]] [t/TAG [MORE_TAG]]`<br> e.g. `searchR i/lettuce tomato, searchR n/salad, searchR t/healthy`
```
Comment (caitlinjee) [inline docs/UserGuide.md]: i think in this example put the ` around each command? instead of around the whole example

**Review by caitlinjee — APPROVED** [review]

File: docs/diagrams/DelectRecipeSequence.puml [hunk 2]
```diff
@@ -0,0 +1,86 @@
+@startuml
```
Comment (TianYong-Goh) [inline docs/diagrams/DelectRecipeSequence.puml]: Put this file in the unused instead

File: docs/diagrams/ListRecipesSequence.puml [hunk 3]
```diff
@@ -0,0 +1,69 @@
+@startuml
```
Comment (TianYong-Goh) [inline docs/diagrams/ListRecipesSequence.puml]: Same as above

**Review by TianYong-Goh — APPROVED** [review]

**Review by oliviajohansen — APPROVED** [review]
lgtm

File: docs/diagrams/DelectRecipeSequence.puml (hunk 2 above)
Comment (jiaqi20) [reply docs/diagrams/DelectRecipeSequence.puml]: okay!

File: docs/UserGuide.md (hunk 1 above)
Comment (jiaqi20) [reply docs/UserGuide.md]: okay!

**Review by daongochieu2810 — APPROVED** [review]

**Pull request merged**